gatsby/content/projects/2018-12-10-matrix-appservice-hangouts.mdx

```diff
@@ -1,15 +1,15 @@
 ---
 layout: projectimage
 title: matrix-appservice-hangouts
-categories: 
+categories:
  - bridge
 description: Hangouts appservice written in Python 3.6 using asyncio.
 author: Cadair
-maturity: Alpha
+maturity: Not actively maintained
```

Comment:
please revert

Reply:
Just to give one example. This bridge is currently broken, requires a patch and no one reacted to this [issue](https://github.com/Cadair/matrix-appservice-hangouts/issues/249) in over 14 months. Does not look like it is maintained or should be featured.